errorLog.csv

```diff
@@ -0,0 +1,14 @@
+1,TypeError,'int' object is not callable,0,,2021-11-07 15:55:34.668983,"[(b'd\x00d\x01\x83\x01F\x00d\x02S\x00', 1)]","['  File ""/usr/local/lib/python3.8/site-packages/IPython/core/interactiveshell.py"", line 3417, in run_code\n    exec(code_obj, self.user_global_ns, self.user_ns)\n', '  File ""<ipython-input-5-9b43eec38a3d>"", line 1, in <module>\n    3(2)\n']"

```

Comment:
If we don't get the Python decompyle to work, we should probably just remove the bytecode column. cc: @kenny-chi